src/extended_json.ts

```diff
@@ -24,11 +24,19 @@ import { Timestamp } from './timestamp';
 
 /** @public */
 export type EJSONOptions = {
-  /** Output using the Extended JSON v1 spec */
+  /**
+   * Output using the Extended JSON v1 spec
+   * @defaultValue `false`
+   * */
```

Comment:
```suggestion
   */
```

This fix in a bunch of places, should always be one star on the last line

```
  /**
   * xx
   */
```